editor/src/Components/DataTable.js

```diff
@@ -177,8 +192,12 @@ const DataTable = ({
                                         name={field}
                                         id={field}
                                         type="text"
-                                        defaultValue={fieldValue}
-                                        onChange={e => handleInputChange(e)}
+                                        value={editValue
+                                          .toString()
+                                          .toUpperCase()}
```

Comment:
updating these to be controlled components allows me to auto capitalize user input as they type